spec/schema.json

```diff
@@ -663,13 +732,13 @@
             "general_partners": {
               "type": "array",
               "items": {
-                "type": "object"
+                "$ref": "#/definitions/PartnerData"
```

Comment:
Wondering if this could be `"$ref": "#/definitions/GeneralPartnerData"` to be consistent with the entry for Limited Partners just below?

Reply:
I named it PartnerData as it's also used in LimitedPartnerData. Essentially LimitedPartnerData is PartnerData plus 3 fields for capital contributions:

`    "LimitedPartnerData": {
      "allOf": [
        {
          "type": "object",
          "properties": {
            "contribution_currency_type": {
              "type": "string"
            },
            "contribution_currency_value": {
              "type": "string"
            },
            "contribution_non_monetary_value": {
              "type": "string"
            }
          }
        },
        {
          "$ref": "#/definitions/PartnerData"
        }
      ]
    },`

Reply:
All good 👍 .